web/test/src/tests/dandisetsPage.test.js

```diff
@@ -26,4 +27,60 @@ describe('dandisets page', () => {
     await expect(page).toMatch(`Contact ${lastName}, ${firstName}`);
     await expect(page).toMatch(`Updated on ${moment(new Date()).format('LL')}`);
   });
+
+  it('search for dandisets', async () => {
+    await registerNewUser();
+
+    const dandisetIdentifierNameMapping = new Map();
+
+    // Create 10 dandisets
```

Comment:
Did you give a lot of thought to scaffolding these tests in python vs js? I don't think it makes much of a difference in the short term, but if we find ourselves writing a lot of these tests we might want to revisit it.

Reply:
No, I tried to keep it as simple as possible and didn't try coming up with a way to scaffold test data. If we do end up writing a lot of these tests, we'll definitely need to consider that, especially in the context of things like publishing and zarr which would **require** some sort of test data setup beforehand, since in normal circumstances those require asset upload via the CLI. I think we still need to decide what the future of tests like these looks like though, so I did the bare minimum for now.

This is one major disadvantage of using js instead of python, where we had direct access to `pytest-django`.